packages/lib/services/KeymapService.ts

```diff
@@ -114,6 +115,7 @@ const defaultKeymapItems = {
 		{ accelerator: 'Ctrl+Alt+3', command: 'switchProfile3' },
 		{ accelerator: 'Ctrl+Alt+N', command: 'openNoteInNewWindow' },
 		{ accelerator: 'Ctrl+M', command: 'toggleTabMovesFocus' },
+		{ accelerator: 'Ctrl+L', command: 'linkToNote' },
```

Comment:
<kbd>ctrl</kbd>-<kbd>l</kbd> may conflict with the default shortcut for `toggleVisiblePanes`:
https://github.com/laurent22/joplin/blob/9251299289527d1e4a0e92502b28a48f4ace9c50/packages/lib/services/KeymapService.ts#L94

Reply:
Oh that's right. I did search for it and was surprised to see that Ctrl+L wasn't in use but obviously I must have missed something here.